lib/types/application/ApplicationBuilder.js

```diff
@@ -138,12 +138,17 @@ class ApplicationBuilder extends AbstractBuilder {
 			});
 		}
 
+		const minificationPattern = ["/**/*.js"];
+		if (["2.6"].includes(project.specVersion)) {
+			const minification = (project.builder && project.builder.minification) || {};
+			this.enhancePatternWithExcludes(minificationPattern, minification.excludes);
```

Comment:
Should the pattern be prefixed with `/resources/` here?
The default pattern for applications is `/**/*.js`, so I'd expect the excludes to be possible for all paths, not just within `/resources/`.
Or do I missing something here? For libraries it totally makese sense as we only process `/resources/`.